pkg/controllers/networkinfo/networkinfo_utils.go

```diff
@@ -191,7 +191,7 @@ func getGatewayConnectionStatus(ctx context.Context, nc *v1alpha1.VPCNetworkConf
 	return gatewayConnectionReady, reason
 }
 
-func updateVPCNetworkConfigurationStatusWithAliveVPCs(ctx context.Context, client client.Client, ncName string, getAliveVPCsFn func(ncName string) []*model.Vpc) {
+func updateVPCNetworkConfigurationStatusWithAliveVPCs(ctx context.Context, client client.Client, ncName string, getAliveVPCsFn func(ncName string) ([]*model.Vpc, error)) {
```

Comment:
this method contains a new return value error, but it is never used?

Reply:
It might look a little bit confusing to read but that is the return value for getAliveVPCsFn instead of updateVPCNetworkConfigurationStatusWithAliveVPCs. The value is used for log as the following
```
                vpcList, err := getAliveVPCsFn(ncName)
		if err != nil {
			log.Error(err, "Failed to get alive VPCs", "Name", ncName)
			return
		}
```

Reply:
maybe i did not explain it clearly, I mean the method gives a return value type, but the value is never used in any return statement in the method logic.